esx_service/utils/auth_api.py

```diff
@@ -299,8 +299,20 @@ def _tenant_create(name, description="", vm_list=None, privileges=None):
     if not description:
         description = ""
 
-    logging.debug("_tenant_create: vms=%s", vms)
+    error_info, tenant_list = get_tenant_list_from_db()
```

Comment:
1. This logic should not be triggered if vm-list is None.
2. The parameter processing sequence in this method is a little bit messy. We should process the parameters one by one clearly: name, vm_list and description.